cli/src/pcluster/resources/imagebuilder/parallelcluster.yaml

```diff
@@ -212,15 +214,10 @@ phases:
               PLATFORM='{{ build.PlatformName.outputs.stdout }}'
 
               if [[ ${!OS} =~ ^centos[7-8]$ ]]; then
-                  if [[ ${!OS} == centos7 ]]; then
-                    yum -y install epel-release
```

Comment:
are we sure that this is not needed for the package `jq` that is installed later?